pkg/kn/commands/types.go

```diff
@@ -70,7 +71,8 @@ func (c *KnParams) GetConfig() (serving.ServingV1alpha1Interface, error) {
 func (c *KnParams) GetClientConfig() clientcmd.ClientConfig {
 	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
 	if len(c.KubeCfgPath) > 0 {
-		loadingRules.ExplicitPath = c.KubeCfgPath
+		paths := filepath.SplitList(c.KubeCfgPath)
```

Comment:
Can we check if the returned slice has 1 element only and then use `ExplicitPath` instead ? The benefit that this would result in an additional check for this file leading to an error if it does not exist ("fail fast").

Reply:
@rhuss Sure. I will add test cases following gotest firstly.

Reply:
What's about the following suggestion, which would also work if the filename contains a path seperator (not very likely but IMO totall valid to have a file which contains a ':' as part of the name):

* Check the fullname if it exists as file via `os.Stat`
  - If yes, pick that name
  - If not:
    * Check if the fullname contains a path seperator.
      - If yes: Return error "Can not find configuration file. %s looks like a combine path. Please use the env var KUBECONFIG if you want to check for multiple configuration files"
      - If no: "Configuration file %s can not be found"

Does this make sense ?

Reply:
+1 on improving the error message returned `"Config file '%s' could not be found. For configuration lookup path please use the env variable KUBECONFIG`